honeycomb-core/src/twomap.rs

```diff
@@ -1117,14 +1111,8 @@ impl<const N_MARKS: usize> TwoMap<N_MARKS> {
                         stretch!(self, lhs_dart_id, b1rid);
                     }
                     SewPolicy::StretchAverage => {
-                        let new_lvertex = [
-                            (lvertex[0] + b1_rvertex[0]) / 2.0,
-                            (lvertex[1] + b1_rvertex[1]) / 2.0,
-                        ];
-                        let new_rvertex = [
-                            (rvertex[0] + b1_lvertex[0]) / 2.0,
-                            (rvertex[1] + b1_lvertex[1]) / 2.0,
-                        ];
+                        let new_lvertex = (lvertex + b1_rvertex) / T::from(2.0).unwrap();
+                        let new_rvertex = (rvertex + b1_lvertex) / T::from(2.0).unwrap();
```

Comment:
Perhaps adding a mid_vertex function or something like this (if useful elsewhere).

Reply:
There are actually a lot of places where I'm doing this, I've added a function in 1fdda5b